at_lookup/example/README.md

```diff
@@ -0,0 +1,13 @@
+<img src="https://atsign.dev/assets/img/@developersmall.png?sanitize=true">
```

Comment:
A lot of what's here should be back in the README.md because it's about the package rather than the example.

Take a look at [archetype-pubdev](https://github.com/atsign-foundation/archetype-pubdev) for some examples. Also we're switching to the new @ platform logo.

Reply:
Updated the README.md content and the @platform logo